pkgs/by-name/ob/objfw/package.nix

```diff
@@ -30,6 +32,47 @@ clangStdenv.mkDerivation (finalAttrs: {
 
   doCheck = true;
 
+  passthru.tests = {
+    build-hello-world = clangStdenv.mkDerivation {
```

Comment:
Might be nice in a different file that gets referenced here to not pollute the package.nix, but the derivation is pretty small so it's not too bad.

Reply:
Ok, I will make that change in a future PR. Thanks for merging!